src/core/store/common/tokensByAsset.selectors.ts

```diff
@@ -1,44 +1,83 @@
 import { createSelector, current } from '@reduxjs/toolkit';
-import { memoize } from 'lodash';
+import { memoize, sortBy, sortedUniq, uniqWith, isEqual } from 'lodash';
 
 import { getConfig } from '@config';
 import { TokenView } from '@types';
 import { testTokens } from '@src/config/constants';
 
 import { VaultsSelectors } from '../modules/vaults/vaults.selectors';
-import { TokensSelectors } from '../modules/tokens/tokens.selectors';
 import { AppSelectors } from '../modules/app/app.selectors';
 import { WalletSelectors } from '../modules/wallet/wallet.selectors';
-import { createToken } from '../modules/tokens/tokens.selectors';
+import { TokensSelectors, createToken } from '../modules/tokens/tokens.selectors';
 import { NetworkSelectors } from '../modules/network/network.selectors';
+// import { Token } from 'graphql';
 
 const { selectVaultsMap } = VaultsSelectors;
-const { selectTokensMap, selectTokensUser } = TokensSelectors;
+const { selectTokenAddresses, selectSupportedTokens, selectSupportedTokensMap, selectTokensMap, selectTokensUser } =
+  TokensSelectors;
 const { selectServicesEnabled } = AppSelectors;
 const { selectCurrentNetwork } = NetworkSelectors;
 const { selectWalletNetwork } = WalletSelectors;
 
 export const selectDepositTokenOptionsByAsset = createSelector(
-  [selectTokensMap, selectTokensUser, selectServicesEnabled, selectWalletNetwork],
-  (tokensMap, tokensUser, servicesEnabled, currentNetwork) =>
+  [
+    selectSupportedTokens,
+    selectSupportedTokensMap,
+    selectTokenAddresses,
+    selectTokensMap,
+    selectTokensUser,
+    selectServicesEnabled,
+    selectWalletNetwork,
+  ],
+  (supportedTokens, supportedTokensMap, tokenAddresses, tokensMap, tokensUser, servicesEnabled, currentNetwork) =>
     memoize((assetAddress?: string): TokenView[] => {
-      console.log('selectDepositTokenOptionsByAsset', currentNetwork, tokensMap, testTokens);
+      console.log('TokenService selectDepositTokenOptionsByAsset', currentNetwork, tokensMap, testTokens);
+      const { userTokensMap, userTokensAllowancesMap } = tokensUser;
       if (currentNetwork === 'goerli') {
+        // TODO: fill in token values appropriately with values from subgraph
```

Comment:
@kibagateaux I commented out this code which was how I was testing the functionality worked using the `goerli` subgraph when on the goerli network. Right now, if you are on `goerli` network you will just see the `testTokens` from the `constants` file in the modals. You have to be on `mainnet` to see all the supported tokens from the subgraph.